tests/conftest.py

```diff
@@ -439,28 +436,29 @@ def sql_engine():
     return create_engine("sqlite:///:memory:")
 
 
-@pytest.fixture()
+@pytest.fixture
 def init_sql_repo(tmp_sqlite):
```

Comment:
Now the `init_sql_repo` fixture will also clean the SQLite database based on the `tmp_sqlite` path